### ISSUE TITLE
LabVIEW Examples in latest release (1.6.7) are saved in LabVIEW 2022 Q3

### ISSUE THREAD

**Issue by jasonrorr** [tmytek/bbox-api#20]
## Description
The latest release of the TMYTEK bbox-api library has all of its examples saved in LabVIEW 2022 Q3. For greater application and development accessibility and compatibility, the examples should be saved for an earlier version of LabVIEW.

The documentation/readme for this library states that LabVIEW 2021 is the tested/prerequisite version of LabVIEW for the API examples, which is no longer the case. This affects all of the example LabVIEW project files and VIs, not just the BBox One examples.

This can be fixed by simply opening the projects in LabVIEW 2022 Q3 and saving them for an earlier version of LabVIEW to maintain backwards compatibility

![image](https://user-images.githubusercontent.com/42351495/232792365-e4b3d23a-2f4b-4778-96b4-d57c6f344303.png)

![image](https://user-images.githubusercontent.com/42351495/232792515-4e48cd2a-5b64-4292-b610-743fb6abfa87.png)

## Steps to Reproduce
1. Download the latest release of the TMYTEK API Examples
2. Attempt to open any of the LabVIEW project files or VIs in LabVIEW 2021
3. LabVIEW will report an error that the file version is later than the current LabVIEW version

![image](https://user-images.githubusercontent.com/42351495/232791733-294ff9f8-7396-4a4a-909a-96186b16b607.png)

## Expected Behavior
The example projects and VIs should open in LabVIEW 2021

## Current Behavior
The example projects and VIs are saved in LabVIEW 2022 Q3, which means they are incompatible with LabVIEW 2021

### BBox API Version
1.6.7

**Comment by jasonrorr** [comment]
@RaymondSuTMY thanks for your help!
